requirements.txt

```diff
@@ -1,7 +1,12 @@
 # Note: this does not include arcgis, which needs to be installed manually when using virtualenv.
 # Use the following command: pip install arcgis --no-deps
 
-bs4==0.0.1
-python-dateutil==2.8.1
-requests==2.22.0
-url-normalize==1.4.1
\ No newline at end of file
+beautifulsoup4==4.9.3
+ntlm-auth==1.5.0
+python-dateutil==2.8.2
+requests==2.26.0
+requests-toolbelt==0.9.1
+requests_ntlm==1.1.0
+six==1.16.0
+url-normalize==1.4.3
+ujson==4.1.0
```

Comment:
The instructions for installing `arcgis` with minimum dependencies seem to have changed. They want you to specify dependencies yourself, including `ntlm-auth`, `requests` (which we use already), `requests-toolbelt`, `requests_ntlm`, `six`, and `ujson`. Some of these dependencies seem to be out of date, especially the `ntlm`-related ones. This seems to work though.